charts/kube-prometheus-stack/templates/prometheus/rules-1.14/windows.node.rules.yaml

```diff
@@ -0,0 +1,112 @@
+{{- /*
+Generated from 'windows.node.rules' group from https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/kubernetesControlPlane-prometheusRule.yaml
```

Comment:
I explained the reason why it can't be created via the hack scripts in my PR description. Also happy to change the header if you tell me what you expect it to look like...